tests/platform_tests/conftest.py

```diff
@@ -86,14 +117,28 @@ def service_time_check(message, status):
         timestamps[status] = time
         service_restart_times.update({service_name: service_dict})
 
+    reboot_time = "N/A"
     for message in messages:
+        if re.search(reboot_pattern, message):
+            reboot_time = datetime.strptime(message.split(duthost.hostname)[0].strip(), FMT).strftime(FMT)
```

Comment:
Same datetime conversion question. Is this equivalent to `reboot_time = message.split(duthost.hostname)[0].strip()`

Reply:
Replied above.